src/Microsoft.Health.Operations.Functions.UnitTests/Management/PurgeOrchestrationInstanceHistoryTests.cs

```diff
@@ -43,14 +46,57 @@ public PurgeOrchestrationInstanceHistoryTests()
     [Theory]
     [InlineData(0)]
     [InlineData(12)]
-    public async Task GivenNoOrchestrationInstances_WhenPurgeCompletedDurableFunctionsHistory_ThenNoOrchestrationsPurgedAsync(int deleted)
+    public async Task GivenOrchestrationInstances_WhenPurgeCompletedDurableFunctionsHistory_ThenOrchestrationsPurgedAsync(int count)
     {
+        var instanceId = Guid.NewGuid().ToString();
+
+        var durableOrchestrationState = Enumerable.Repeat(new DurableOrchestrationStatus { InstanceId = instanceId }, count);
+
+        _durableClient
+            .ListInstancesAsync(Arg.Is<OrchestrationStatusQueryCondition>(condition => AreConditionEqual(condition)), Arg.Any<CancellationToken>())
```

Comment:
nit: 
```suggestion
            .ListInstancesAsync(Arg.Is<OrchestrationStatusQueryCondition>(AreConditionEqual), Arg.Any<CancellationToken>())
```

Reply:
Not sure if that works, it cannot convert a method group to a condition